datasets/census_bureau_acs/_images/run_csv_transform_kub/csv_transform.py

```diff
@@ -0,0 +1,550 @@
+        "B15001030": "male_45_64_high_school",
+        "B15001031": "male_45_64_some_college",
+        "B15001032": "male_45_64_associates_degree",
+        "B15001033": "male_45_64_bachelors_degree",
+        "B15001034": "male_45_64_graduate_degree",
+        "B12005001": "pop_15_and_over",
+        "B12005002": "pop_never_married",
+        "B12005005": "pop_now_married",
+        "B12005008": "pop_separated",
+        "B12005012": "pop_widowed",
+        "B12005015": "pop_divorced",
+        "B08134001": "commuters_16_over",
+        "B08134002": "commute_less_10_mins",
+        "B08303003": "commute_5_9_mins",
+        "B08303004": "commute_10_14_mins",
+        "B08303005": "commute_15_19_mins",
+        "B08303006": "commute_20_24_mins",
+        "B08303007": "commute_25_29_mins",
+        "B08303008": "commute_30_34_mins",
+        "B08303009": "commute_35_39_mins",
+        "B08303010": "commute_40_44_mins",
+        "B08134008": "commute_35_44_mins",
+        "B08303011": "commute_45_59_mins",
+        "B08134010": "commute_60_more_mins",
+        "B08303012": "commute_60_89_mins",
+        "B08303013": "commute_90_more_mins",
+        "B08135001": "aggregate_travel_time_to_work",
+        "B19001002": "income_less_10000",
+        "B19001003": "income_10000_14999",
+        "B19001004": "income_15000_19999",
+        "B19001005": "income_20000_24999",
+        "B19001006": "income_25000_29999",
+        "B19001007": "income_30000_34999",
+        "B19001008": "income_35000_39999",
+        "B19001009": "income_40000_44999",
+        "B19001010": "income_45000_49999",
+        "B19001011": "income_50000_59999",
+        "B19001012": "income_60000_74999",
+        "B19001013": "income_75000_99999",
+        "B19001014": "income_100000_124999",
+        "B19001015": "income_125000_149999",
+        "B19001016": "income_150000_199999",
+        "B19001017": "income_200000_or_more",
+        "B19058002": "households_public_asst_or_food_stamps",
+        "B19059002": "households_retirement_income",
+        "B25064001": "renter_occupied_housing_units_paying_cash_median_gross_rent",
+        "B25076001": "owner_occupied_housing_units_lower_value_quartile",
+        "B25077001": "owner_occupied_housing_units_median_value",
+        "B25078001": "owner_occupied_housing_units_upper_value_quartile",
+        "B07204001": "population_1_year_and_over",
+        "B07204004": "different_house_year_ago_same_city",
+        "B07204007": "different_house_year_ago_different_city",
+        "B26001001": "group_quarters",
+        "B08014002": "no_car",
+        "C24060004": "sales_office_employed",
+        "C24060002": "management_business_sci_arts_employed",
+        "B23006001": "pop_25_64",
+        "B23006023": "bachelors_degree_or_higher_25_64",
+        "B11001007": "nonfamily_households",
+        "B11001002": "family_households",
+        "B25035001": "median_year_structure_built",
+    }
+
+    state_code = {
```

Comment:
We can the same here (separate file loaded)

Reply:
created separate json file